tests/test_utils.py

```diff
@@ -219,16 +218,14 @@ def test_str_to_unicode():
     s = ["test1", "test2", b"test3"]
     exp_answer = ["test1", "test2", "test3"]
     assert_array_equal(utils.str_to_unicode(s), exp_answer)
-
     # set of str
     s = {"test1", "test2", b"test3"}
     exp_answer = {"test1", "test2", "test3"}
-    assert (utils.str_to_unicode(s), exp_answer)
-
+    assert_equal(utils.str_to_unicode(s), exp_answer)
```

Comment:
Putting back the blank line for readability 
```suggestion
    assert_equal(utils.str_to_unicode(s), exp_answer)

```